crates/control/src/led_status.rs

```diff
@@ -207,30 +207,40 @@ impl LedStatus {
         blink_state: bool,
         current_maximum_temperature: f32,
     ) -> Ear {
-        if filter_whistle_detected {
-            return Ear::full_ears(1.0);
-        }
+        let temperature_level_one = 76.0;
+        let temperature_level_two = 80.0;
+        let temperature_level_three = 90.0;
+        // values, at which the stiffness gets reduced
```

Comment:
Why is this comment below of the constants?

Reply:
Done